pkg/sentry/kernel/task_run.go

```diff
@@ -132,6 +133,7 @@ func (t *Task) doStop() {
 // +stateify savable
 type runApp struct{}
 
+// +noescape
```

Comment:
I must be confused what the noescape package means by "recursive" analysis, because this function recursively includes basically the entire sentry codebase via syscall implementations, so it _definitely_ has escapes.

Reply:
The default +noescape tag will limit it's analysis only to the local scope. If you explicitly name a type, it will be recursive by default, e.g.


+checkescape:hard,local <- default, equivalent to +checkescape

+checkescape:hard <- recursively check for any hard escapes.
+checkescape:all <- recursively check for any hard & soft escapes.
+checkescape:all,local <- check for any hard or soft local escapes.